src/main/java/org/openpnp/machine/reference/driver/GrblDriver.java

```diff
@@ -51,92 +49,84 @@
 /**
  * TODO: Consider adding some type of heartbeat to the firmware.  
  */
+
 public class GrblDriver extends AbstractSerialPortDriver implements Runnable {
+
```

Comment:
I was referring to vacPumpPin and invertVacpump. If they are not needed then they should be removed.